.github/CODEOWNERS

```diff
@@ -1,13 +1,13 @@
-* @aws-amplify/amplify-support-eng
-/src/pages/** @aws-amplify/amplify-support-eng
-/src/pages/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng
-/src/pages/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng
-/src/pages/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng
-/src/fragments/** @aws-amplify/amplify-support-eng
-/src/fragments/**/android @aws-amplify/amplify-android @aws-amplify/amplify-support-eng
-/src/fragments/**/flutter @aws-amplify/amplify-flutter @aws-amplify/amplify-support-eng
-/src/fragments/**/ios @aws-amplify/amplify-ios @aws-amplify/amplify-support-eng
-/src/fragments/**/js  @aws-amplify/amplify-js @aws-amplify/amplify-support-eng
-/src/fragments/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng
-/src/fragments/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng
-/src/fragments/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng
+* @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/** @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/** @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/android @aws-amplify/amplify-android @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/flutter @aws-amplify/amplify-flutter @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/ios @aws-amplify/amplify-ios @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/js  @aws-amplify/amplify-js @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
```

Comment:
Just noticed that, the `guides` routes already has `aws-amplify/developer-advocates` :
- `/src/pages/guides`
- `/src/fragments/guides`